Open Judge System/Data/OJS.Data/Migrations/DefaultMigrationConfiguration.cs

```diff
@@ -41,6 +41,57 @@ protected override void Seed(OjsDbContext context)
         }
 
         //// TODO: Add seed with .Any()
+        //protected void SeedCopyOfSubmissionAndCreateTestRuns(OjsDbContext context)
```

Comment:
Is this code going to be used later and soon or maybe we can remove it now, instead of commenting it?